default/templates/default/base-uploader.html

```diff
@@ -12,8 +12,11 @@
 
 {% block body %}
 <div class="response-success alert alert-info hidden">
-    <b>{% trans "Success!" %}</b> result.zip(<span class="file-size"></span>)
-    <a href="javascript:void(0)" class="download" target="_blank">{% trans "Download" %}</a>
+    <b>{% trans "Success!" %}</b>
+    result.zip(<span class="file-size"></span>)
+    <a href="javascript:void(0)" class="download" target="_blank">{% trans "Local download" %}</a>
+    <span>  /  </span>
+    <a href="javascript:void(0)" class="download-drive">{% trans "Upload to Google Drive" %}</a>
```

Comment:
"Save to Google Drive"?

Reply:
Also I suspect this link takes the page to a blank success/error message. Use this link (or a button) to make an AJAX call to the server and show the response in Toucan's interface.